examples/step-67/doc/kind

```diff
@@ -0,0 +1 @@
+techniques
```

Comment:
maybe make it a fluid flow problem instead?

Reply:
Yes, fluid flow is good.